framework/tlvf/test/main.cpp

```diff
@@ -33,11 +36,66 @@ MAPF_INITIALIZE_LOGGER
 
 using namespace mapf;
 
+void dump_buffer(uint8_t *buffer, size_t len)
+{
+    for (size_t i = 0; i < len; i += 16) {
+        std::ostringstream hexdump;
+        for (size_t j = i; j < len && j < i + 16; j++)
+            hexdump << std::hex << std::setw(2) << std::setfill('0') << (unsigned)buffer[j] << " ";
+        LOG(INFO) << hexdump.str();
+    }
+}
+
+int test_int_len_list()
+{
+    int errors = 0;
+    uint8_t tx_buffer[4096];
+    const uint8_t gTlvMacAddress[6] = {0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff};
+
+    MAPF_INFO(__FUNCTION__ << " start");
+    memset(tx_buffer, 0, sizeof(tx_buffer));
+    CmduMessageTx msg = CmduMessageTx(tx_buffer, sizeof(tx_buffer));
```

Comment:
For this test we don't really need a CMDU, it's sufficient to create the tlvMacAddress with the buffer as argument and call class_swap on it.